modal_proto/api.proto

```diff
@@ -1712,15 +1712,14 @@ message RuntimeOutputMessage {
 }
 
 message RuntimeOutputBatch {
-  string entry_id = 1;
+  int32 batch_index = 1; 
   repeated RuntimeOutputMessage items = 2;
   bool eof = 3;
 }
```

Comment:
In gRPC, it's not a good idea to overwrite fields that need to be backwards compatible. You should add a new field and deprecate the old one. And then, when backwards compatibility isn't an issue, delete the deprecated field. 

```proto
message RuntimeOutputBatch {
  string entry_id = 1 [deprecated=true];
  repeated RuntimeOutputMessage items = 2;
  bool eof = 3;
  int32 batch_index = 4; 
}
```